### DISCUSSION TITLE
Creating your own flight, THEN having Liberation generate the rest?

### DISCUSSION THREAD

**Discussion by jubuttib** [Khopa/dcs_liberation#1580] [Q&A] [answered]
Is it currently possible to create your own flight package, to make sure you get to fly the plane/heli of your choosing, and THEN have Liberation generate all the other ones?

I like playing as mostly "just a pilot", having Liberation automate both ground and aircraft purchses etc., but sometimes I run into one of two situations:

1. There's a specific task I want to accomplish, but it's not among the ones generated for the turn, and all the available planes are taken.
2. There isn't a package with a module I own.

Currently I think all the flight packages are generated automatically at the start of the turn, so I'm somewhat at the mercy of Liberation on whether I get a suitable mission.

Can I do something like delete the generated packages, then create my own, and then have Liberation generate a new set of packages again?

**Comment by DanAlbert** [comment, marked as answer]
> Currently I think all the flight packages are generated automatically at the start of the turn, so I'm somewhat at the mercy of Liberation on whether I get a suitable mission.

Let Liberation plan, then plan your flight. Delete an existing package to free up aircraft if you need to. There's no way to do it the other way around.

If by "suitable" mission you just mean you don't want to fly barcap or whatever, go into the air wing menu, pick your squadron, and disable auto-assignment of the mission types you don't want to fly.

**Comment by jubuttib** [comment]
I meant more along the lines of "modules I own/am proficient in". Thanks for the confirmation, will try to work around the limitations.

Would love to have a feature in the future for being able to first make your package, then have Liberation plan the rest.

**Reply by autumncurve** [reply]
I would also really appreciate a way to set before the campaign is generated, what modules you/your friend owns and the campaign generate based on what you can fly as a result. 

I’ve been playing with a friend who only owns the A10A and I have to start the campaign myself and then order one in specifically before he can start flying.

Am I right in thinking a couple of the campaigns start with only access to a carrier and therefore planes that can take off from carriers? E.g scenic route on Persian gulf. We haven’t been able to work out how he can join in on those.

We’re loving liberation though when we get it going and get him involved. Incredible addition to DCS!

**Reply by DanAlbert** [reply]
You can choose what squadrons are in the game.

**Reply by autumncurve** [reply]
Ah cool, I must have missed that then! I’ll hunt around for it.
